arrow-fx-coroutines/src/test/kotlin/arrow/fx/coroutines/ResourceTest.kt

```diff
@@ -0,0 +1,59 @@
+package arrow.fx.coroutines
+
+import arrow.core.Either
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.shouldBe
+import io.kotest.property.Arb
+import io.kotest.property.arbitrary.int
+import io.kotest.property.checkAll
+
+class ResourceTest : StringSpec({
+
```

Comment:
Should we have some laws-checking tests here? e.g. to verify that composition of `Resource`s obeys the monad laws? I see we do law-checking in the tests for the original arrow-fx `Resource`.

Reply:
Yes, we definitely should! This PR currently doesn't include any instances yet.
Next week I'll be creating the issues, including this, to work towards a first release.